### PR TITLE
New server tab

### PR DESCRIPTION
**Pull request by SaraVieira**
YAY

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3397/commits/e402b36c7a442fc9028284a1d408f7b9670d271d is at https://pr3397.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      >
+        <Input
+          value={value}
+          onKeyDown={onKeyDown}
+          onChange={e => setValue(e.target.value)}
+          required
+          placeholder="Value"
+        />
+      </FormField>
+      <input id="submit" type="submit" style={{ display: 'none' }} />
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx]: Hi! 

Can we add an actual button here: `<Button variant="secondary" type="submit">Add secret</Button>` (that's what I'm doing in #3399)

**Comment by siddharthkp** [comment]
there's something up with the input fields, they are very excited :P

Screenshot 2020-01-27 at 4 08 20 PM

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <Collapsible defaultOpen title="Server Control Panel">
+    <Element padding={2} marginBottom={5}>
+      <Text weight="medium" block marginBottom={2}>
+        Server Sandbox
+      </Text>
+      <Text variant="muted">
+        This sandbox is executed on a server. You can control the server from
+        this panel.
+      </Text>
+      <Element marginTop={5}>
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx]: I would stick to marginBottom unless absolutely necessary

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        name,
+        value,
+      });
+      setName('');
+      setValue('');
+    }
+  };
+
+  const onKeyDown = e => {
+    if (e.key === 'Escape' && props.onEscClicked) {
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx]: Should we just use a form and then skip the `escape` implementation? 🤔

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx]: Changed :)

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx (hunk 3 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx]: I involved danny and we decided to do both, esc still stays for more advanced people but we have the button too :)

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx]: Talked with danny and added it :)

**Comment by SaraVieira** [comment]
> there's something up with the input fields, they are very excited :P

That's an issue we are definitely gonna have to fix with the old codesandbox theme :(

**Comment by siddharthkp** [comment]
> That's an issue we are definitely gonna have to fix with the old codesandbox theme :(

oh that's on the old one, gotcha!

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx (hunk 2 above)
Comment (siddharthkp) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx]: Err not sure if I'm looking at an outdated version, but I see marginTop?

**Pull request merged**